engine/runtime/src/test/scala/org/enso/interpreter/test/instrument/RuntimeStdlibTest.scala

```diff
@@ -245,6 +252,28 @@ class RuntimeStdlibTest
       ("Standard", "Base", TestEdition.testLibraryVersion.toString)
     )
 
+    // check documentation generation
+    failAfter(60.seconds) {
```

Comment:
Shouldn't we simply put a time limit on all tests using the `TimeLimitedTests` mixin? Seems like it could be more robust as we do not really know which test can begin looping (probably need a high-enough time limit to avoid failing due to slowness, but I think even limit like 5-10 minutes could work).

Still may be worth keeping this lower limit for the ones that are more prone to looping.